### PR TITLE
use our own unison-fsmonitor distrib

### PR DESCRIPTION
**Pull request by otakumike**

### CODE REVIEW

File: system/goos/dependencies_darwin.go [hunk 1]
```diff
@@ -50,68 +53,37 @@ func CheckDockersync() {
...
-			return err
-		}
-
-		fs.Remove(ut)
-		fs.RemoveAll(td)
-
-		fmt.Println("  √  unison-fsmonitor")
-
-		return nil
+	_, err = utils.CommandSubSplitOutput("pip", "install", "watchdog")
```
Comment (yjimk) [inline system/goos/dependencies_darwin.go]: Still need to use pip?

**Comment by garrettw** [comment]
Interesting. I'm curious as to how your distro compares to upstream, but this sounds like a potentially truly differentiating factor for Tokaido in its market.

File: system/goos/dependencies_darwin.go (hunk 1 above)
Comment (otakumike) [reply system/goos/dependencies_darwin.go]: yup still do unfortunately

**Comment by otakumike** [comment]
> Interesting. I'm curious as to how your distro compares to upstream, but this sounds like a potentially truly differentiating factor for Tokaido in its market.

Hey @garrettw  :) 

Are you referring to the upstream unison-fsmonitor? If so, thankfully unison-fsmonitor is nice and static, so we're in line with upstream and hopefully will stay that way until a newer release of Unison itself comes out. That happens rarely, so we're not actually sure what will happen when it does.

**Pull request merged**